x/tx/signing/std/handler_map.go

```diff
@@ -0,0 +1,28 @@
+package std
```

Comment:
std just means standard or default. `default` is a reserved keyword but I wanted to do that
We can't put it in `signing` because this depends on all the other sign modes and all sign modes depend on `signing`.